### PR TITLE
Add TCB 59 about jOOQ

### PR DESCRIPTION
**Pull request by mosabua**
Also fyi @lukaseder

### CODE REVIEW

File: _episodes/59.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+[Lukas](https://twitter.com/lukaseder) is recognized as a Java Champion and
+well-known as a very active member of the Java community. We chat about his
+history and involvement in the community of Java and related open source
+projects, and how it lead to [jOOQ and his company data
+geekery](https://www.jooq.org/). Lukas also briefly talks about other products.
+
+## jOOQ
+
+jOOQ stands for jOOQ Object Oriented Querying (jOOQ). It generates Java code
```
Comment (colebow) [inline _episodes/59.md:95]:
```suggestion
jOOQ stands for Java Object Oriented Querying (jOOQ). It generates Java code
```
I assume? Unless it really is recursion all the way down

File: _episodes/59.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+All editions of jOOQ since the 3.19 release include support for Trino. The
+level of support depends on the used catalog and connector, and further
+Trino-specific enhancements are in progress.
+
+<a href="{{site.url}}/ecosystem/add-on.html#jooq">
+  <img src="{{site.url}}/assets/images/logos/jooq.png">
+</a>
+
+In our conversation and demo session with Lukas we cover all the following
```
Comment (colebow) [inline _episodes/59.md:107]:
```suggestion
In our conversation and demo session with Lukas, we cover all the following
```

File: _episodes/59.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+aspects and a few other topics:
+
+* What is jOOQ?
+* What motivated the creation of jOOQ.
```
Comment (colebow) [inline _episodes/59.md:111]:
```suggestion
* What motivated the creation of jOOQ?
```

**Review by colebow — APPROVED** [review]
Couple nits but looks good

File: _episodes/59.md (hunk 1 above)
Comment (mosabua) [reply _episodes/59.md:95]: Its recursion ...

Comment (lukaseder) [reply _episodes/59.md:95]: Yes, see also: https://www.jooq.org/how-to-pronounce-jooq

**Pull request merged**